MAUI/Avatar-view/Getting-Started.md

```diff
@@ -7,19 +7,37 @@ control: SfAvatarView
 documentation: ug
 ---
 
-# Getting Started With .NET MAUI Avatar View (SfAvatarView)
+# Getting Started with .NET MAUI Avatar View
 
 ## Creating an application using the .NET MAUI Avatar View
 
-This section explains the steps required to work with the [SfAvatarView](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Core.SfAvatarView.html?tabs=tabid-1) control for .NET MAUI.
+This section guides you through setting up and configuring a [AvatarView](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Core.SfAvatarView.html?tabs=tabid-1) in your .NET MAUI application. Follow the steps below to add a basic Avatar View to your project.
 
-## Adding .NET MAUI Avatar View reference
+## Prerequisites
 
-The Syncfusion .NET MAUI controls are available in [Nuget.org](https://www.nuget.org/). To add [.NET MAUI Avatar View](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Core.SfAvatarView.html?tabs=tabid-1) to your project, open the NuGet package manager in Visual Studio, search for [Syncfusion.Maui.Core](https://www.nuget.org/packages/Syncfusion.Maui.Core/) and then install it.
+Before proceeding, ensure the following are in place:
+1. Install [.NET 7 SDK](https://dotnet.microsoft.com/en-us/download/dotnet/7.0) or later.
+1. Set up a .NET MAUI environment with Visual Studio 2022 (v17.3 or later) or VS Code. For VS Code users, ensure that the .NET MAUI workload is installed and configured as described [here](https://github.com/dotnet/maui/wiki/VS-Code-setup).
 
-## Handler registration 
+To get start quickly with our .NET MAUI Avatar View, you can check the below video.
 
- In the MauiProgram.cs file, register the handler for Syncfusion core.
+{% youtube "https://www.youtube.com/watch?v=XrTvIgeeUm0" %}
+
+## Step 1: Create a New MAUI Project
+
+1. Open Visual Studio or VS Code.
```

Comment:
Launch Visual Studio or VS Code.
Navigate to File > New > Project, then select the .NET MAUI App template.
Name the project and choose a location, then click Create.

Reply:
changes updated